ui/consoleKeyboardOptions.jsp

```diff
@@ -0,0 +1,24 @@
+<%--
+     Licensed to the Apache Software Foundation (ASF) under one
+     or more contributor license agreements.  See the NOTICE file
+     distributed with this work for additional information
+     regarding copyright ownership.  The ASF licenses this file
+     to you under the Apache License, Version 2.0 (the
+     "License"); you may not use this file except in compliance
+     with the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing,
+     software distributed under the License is distributed on an
+     "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+     KIND, either express or implied.  See the License for the
+     specific language governing permissions and limitations
+     under the License.
+     --%>
+<option value=""></option>
+<option value="us"><fmt:message key="label.standard.us.keyboard" /></option>
+<option value="uk"><fmt:message key="label.uk.keyboard" /></option>
+<option value="jp"><fmt:message key="label.japanese.keyboard" /></option>
+<option value="sc"><fmt:message key="label.simplified.chinese.keyboard" /></option>
+<option value="fr">French AZERTY keyboard</option>
```

Comment:
That will be better if the French label use a localization key from messages.properties like the other languages.